src/app/forms/services/dynamic-form.types.ts

```diff
@@ -122,7 +128,7 @@ export interface FormNode {
   enableDecimals?: boolean;
   nestingLevel?: number;
   groups?: string[];
-  component?: typeof BaseControlComponent;
+  component?: typeof BaseControlComponent | typeof AdrTankDetailsSubsequentInspectionsComponent;
```

Comment:
Are you sure you still need this, the AdrTankDetails component is already extending the basecontrol component through the custom control component so this shouldn't be necessary (this also creates a dependency cycle).